### ISSUE TITLE
server.wait() is not compatibile with Try Except

### ISSUE THREAD

**Issue by markub3327** [deepmind/reverb#59]
Hello,

  I have standalone code for the DB server instance that is not responding on KeyboardInterrupt. I need to store a checkpoint of DB when I'm done with experiments. The DB is not responding on any key shortcut and I had to take it down by `ps -all` and `kill`, but that lost all data without saving checkpoint.

```python
self.server = reverb.Server(....)

self.client = reverb.Client("localhost:8000")

try:
    self.server.wait()
except KeyboardInterrupt:
    print("Terminated by user 👋👋👋")
finally:
    # Save database
    self.client.checkpoint()
```

Thanks.

**Comment by acassirer** [comment]
Hey Martin,

Thanks for reporting this. This feature is currently not implemented in Reverb as per [this line](https://github.com/deepmind/reverb/blob/master/reverb/cc/platform/default/server.cc#L75). I'll try to prioritise this feature for you.

**Comment by markub3327** [comment]
Thanks a lot, I will so happy for new release.

**Comment by acassirer** [comment]
I added support for SIGINT handling with [this commit](https://github.com/deepmind/reverb/commit/a9fcc3550a430a485a93b2688f30b1bdd7fe7127). This should be merged into nightly by tomorrow and included in the next stable version which will be released when Tensorflow 2.6 is released.

Cheers,
